### ISSUE TITLE
add metric to tooltip without it being charted (visible only in tooltip) 

### ISSUE THREAD

**Issue by daniel10012** [apache/superset#13446]
For example in Barcharts, I am looking to be able to see a metric by hovering on the barchart, but without it being graphed

![image](https://user-images.githubusercontent.com/10360991/109882818-735e9500-7c48-11eb-81d5-59f064582b55.png)

Here I would like the Count not to appear in the bars but only in tooltip

**Comment by daniel10012** [comment]
@junlincc

**Comment by junlincc** [comment]
@daniel10012 try going to Customize tab and uncheck the Bar value box? 

<img width="804" alt="Screen Shot 2021-03-08 at 9 04 15 PM" src="https://user-images.githubusercontent.com/67837651/110421189-db9ae580-8051-11eb-8cd2-af6c6f0707c3.png">

**Comment by daniel10012** [comment]
@junlincc this just adds/removes the bar value at the top but not the fact that it is graphed ?

**Comment by junlincc** [comment]
hm I don't understand what you mean by ‘in the graph’😳

**Comment by villebro** [comment]
@junlincc I think the idea is to add metrics that only show up on the tooltip, not on the chart (this is available e.g. in Tableau). This is fairly simple to implement (good starter task).

**Comment by ValentinC-BR** [comment]
This would be a great enhancement !

This could be done in the UI via an additional section "tooltip" with a "metrics" input for instance.

**Comment by junlincc** [comment]
@ValentinC-BR right, that will would in Customize tab, assuming we are keeping the current tab layout.

**Comment by mihir174** [comment]
We could add an additional section for tooltip metadata like this - 

<img width="288" alt="Line Chart - Data (Default)" src="https://user-images.githubusercontent.com/64227069/111694049-9a8f9780-87ee-11eb-94e0-7a2c45cf469f.png">

But I think we should think about chart customization more holistically, and identify other additional input fields we might want to have eg. label, color etc like Tableau
